src/pkcs11/pkcs11-object.c

```diff
@@ -281,8 +281,15 @@ C_GetAttributeValue(CK_SESSION_HANDLE hSession,	/* the session's handle */
 		}
 	}
 
-out:	sc_log(context, "C_GetAttributeValue(hSession=0x%lx, hObject=0x%lx) = %s",
+out:
+	name = lookup_enum ( RV_T, rv );
```

Comment:
couple of bogus whitespace characters
```suggestion
	name = lookup_enum(RV_T, rv);
```